lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -108,10 +117,10 @@ export default async function({workspace, taskUtil, options = {}}) {
 			log.verbose("Writing flexibility changes bundle");
 			return workspace.write(resource);
 		}));
-		// Add the sap.ui.fl dependency if a bundle has been created
-		if (processedResources.length > 0) {
-			await updateFLdependency();
-		}
+		// Add the sap.ui.fl dependency if a bundle has been created and flag if a bundle was created
+		bBundleCreated = processedResources.length > 0;
+		await updateManifestWithFlDependencyAndFlexBundleFlag(bBundleCreated);
+		manifestUpdated = true;
```

Comment:
Calling `updateManifestWithFlDependencyAndFlexBundleFlag` here and having a `manifestUpdated` flag is not needed in case the function should be always called anyways, right?

It could just be called at the end of the function without a condition.